spec/negotiation.html

```diff
@@ -327,21 +327,32 @@ <h1>CoerceOptionsToObject ( _options_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-getoption" aoid="GetOption">
-      <h1>GetOption ( _options_, _property_, _type_, _values_, _fallback_ )</h1>
-
-      <p>
-        The abstract operation GetOption extracts the value of the property named _property_ from the provided _options_ object, converts it to the required _type_, checks whether it is one of a List of allowed _values_, and fills in a _fallback_ value if necessary. If _values_ is *undefined*, there is no fixed set of values and any is permitted.
-      </p>
-
+    <emu-clause id="sec-getoption" type="abstract operation">
+      <h1>
+        GetOption (
+          _options_: an Object,
+          _property_: a property key,
+          _type_: *"boolean"* or *"number"* or *"string"*,
+          _values_: *undefined* or a List of ECMAScript language values,
+          _default_: ~required~ or an ECMAScript language value,
+        )
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It extracts the value of the specified property of _options_, converts it to the required _type_, checks whether it is allowed by _values_ if _values_ is not *undefined*, and substitutes _default_ if the value is *undefined*.</dd>
+      </dl>
       <emu-alg>
-        1. Assert: Type(_options_) is Object.
         1. Let _value_ be ? Get(_options_, _property_).
-        1. If _value_ is *undefined*, return _fallback_.
-        1. Assert: _type_ is *"boolean"* or *"string"*.
+        1. If _value_ is *undefined*, then
+          1. If _default_ is ~required~, throw a *RangeError* exception.
+          1. Return _default_.
         1. If _type_ is *"boolean"*, then
           1. Set _value_ to ! ToBoolean(_value_).
-        1. If _type_ is *"string"*, then
+        1. If _type_ is *"number"*, then
+          1. Set _value_ to ? ToNumber(_value_).
+          1. If _value_ is *NaN*, throw a *RangeError* exception.
+        1. Else,
+          1. Assert: _type_ is *"string"*.
```

Comment:
Why are these strings and not spec enums?

Reply:
I was just about to open a discussion on that topic. ECMA-402 currently uses `*"boolean"*`/`*"number"*`/`*"string"*` strings, Temporal currently uses direct `Boolean`/`String`/`Number` [Type](https://tc39.es/ecma262/multipage/ecmascript-data-types-and-values.html#sec-ecmascript-data-types-and-values) references, and the more conventional approach would be a spec enum like `~boolean~`/`~number~`/`~string~`. The string approach is obviously dispreferred, but how do we feel about direct Type vs. spec enum?